condDatasetSubmitter.py

```diff
@@ -704,6 +710,14 @@ def printInfo(options):
 #-------------------------------------------------------------------------------
 
 if __name__ == "__main__":
+  #Raise an error if couchID files exist
+  p = subprocess.Popen("ls", stdout=subprocess.PIPE, shell=True)
+  out = p.stdout.read().strip()
+  newlist = out.split('\n')
+  substring = ".couchID"
+  for object in newlist:
+    if substring in object:
+      raise ValueError("couchID file exists, please remove it");
```

Comment:
@franzoni @bajarang thanks for implementing this. Do you think that also removing the ".couchID" file from the $PWD would help, other than raising an error?